invenio_rdm_records/resources/serializers/schemas.py

```diff
@@ -79,3 +86,17 @@ def get_contributors(self, obj):
         return [
             c["person_or_org"]["name"] for c in obj["metadata"].get("contributors", [])
         ] or missing
+
+    @post_dump(pass_original=True)
+    def post_dump_pipeline(self, data, original, many, **kwargs):
+        """Defines sequence of post dump steps."""
+        result = deepcopy(data)
```

Comment:
I'm not a big fan of side-effects, but the `post_dump` API for marshmallow kind of assumes it, i.e. `data` is modified and returned from the function. In that sense, we can follow the same principle and just re-assign `data` on each processor call.